README.md

```diff
@@ -1,159 +1,173 @@
+This overhaul resulted in several of Console's key components being available for re-use in any terminal implementation on Windows. These components include a new DirectWrite-based text layout and rendering engine, a text buffer capable of storing both UTF-16 and UTF-8, and a VT parser/emitter, and more.
 
-## Building a new terminal
+## Building the new terminal
 
-When we started building the new terminal application, we explored and evaluated several approaches and technology stacks. We ultimately decided that our goals would be best met by sticking with C++ and sharing the aforementioned modernized components, placing them atop the modern Windows application platform and UI framework.
+When we started building the new Windows Terminal application, we explored and evaluated several approaches and technology stacks. We ultimately decided that our goals would be best met by sticking with C++ and reusing several of the aforementioned modernized components. Further, we realized that this would allow us to build much of the Terminal's core itself as a reusable Windows UI control that others can incorporate into their applications.
 
-Further, we realized that this would allow us to build the terminal's renderer and input stack as a reusable Windows UI control that others can incorporate into their applications.
+The result of this work is contained within this repo and delivered as the Windwos Terminal application you can dowload from the Microsoft Store, or directly from this repo.
 
-# FAQ
+---
 
-## Where can I download Windows Terminal?
+## Resources
+For more information about Windows Terminal, you may find some of these resources useful and interesting:
+* [Command-Line Blog](https://devblogs.microsoft.com/commandline)
+* [Command-Line Backgrounder Blog Series]()
+* Windows Terminal Launch: [Terminal "Sizzle Video"](https://www.youtube.com/watch?v=8gw0rXPMMPE&list=PLEHMQNlPj-Jzh9DkNpqipDGCZZuOwrQwR&index=2&t=0s)
+* Windows Terminal Launch: [Build 2019 Session](https://www.youtube.com/watch?v=KMudkRcwjCw)
+* Run As Radio: [Show 645 - Windows Terminal with Richard Turner](http://www.runasradio.com/Shows/Show/645)
+* Azure Devops Podcast: [Episode 54 - Kayla Cinnamon and Rich Turner on DevOps on the Windows Terminal](http://azuredevopspodcast.clear-measure.com/kayla-cinnamon-and-rich-turner-on-devops-on-the-windows-terminal-team-episode-54)
+---
 
-The latest release of Windows Terminal can be downloaded from [this repo's Releases](https://github.com/microsoft/terminal/releases) page or [from the Microsoft Store](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701)
+## FAQ
 
-## I built and ran the new Terminal, but I just get a blank window app!
+## I built and ran the new Terminal, but I just see a blank window app
```

Comment:
This can probably be toast. I'm pretty sure we put in a dialog box that pops up and stops you from running on the wrong architecture in #1648.

Reply:
FAQ has been re-worked.